extensions-contrib/prometheus-emitter/src/main/resources/defaultMetrics.json

```diff
@@ -0,0 +1,128 @@
+{
+  "query/time" : { "dimensions" : ["dataSource", "type"], "type" : "timer", "conversionFactor": 1000.0, "help":  "Seconds taken to complete a query."},
```

Comment:
Are these conversion a good idea?

It would mean that these metrics will be slightly different from how they are described in this documentation. https://druid.apache.org/docs/latest/operations/metrics.html

Reply:
https://prometheus.io/docs/practices/naming/#base-units
It's coming from prometheus common practice. 

cc @michaelschiff

Reply:
Okay. I guess there are tradeoffs with either choices. Maybe a good way to do is that for those converted, we can just put the unit in the prometheus names. Otherwise, if we refer to the druid metrics doc, we would find the name of the metrics to be documented in a different unit.